components/AdminLessonsSideBar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```

Comment:
move this file to `components/admin/lessonSideBar`

Reply:
hopefully there is an admin folder inside `components`

Reply:
since this is not really reusable and not too long, does it make sense to put this inside `pages/admin/lessons.tsx`

Reply:
I think it would be better to make an `lessons` folder like
`components/admin/lessons/lessonSideBar`
That way, if the other admin pages need to make un-reusable components, another folder can be created inside
`components/admin` to store the files.

For example:
`components/admin/users` <-- here, all components needed for the admin users page can go here